src/logstash/scripts/production_locations/geocode.rb

```
@@ -0,0 +1,39 @@
+  if results.empty?
+    return [event]
+  end
+
+  location_type = results[0]['geometry']['location_type']
+  formatted_address = results[0]['formatted_address']
+
+  event.set('geocoded_location_type', location_type)
+  event.set('geocoded_address', formatted_address)
+
+  return [event]
+end
```

RuboCop (1.76.1)

[convention] Cyclomatic complexity for `filter` is too high. [8/7]

(Metrics/CyclomaticComplexity)